# Hi! I'm Kameron
### I'm a postdoctoral fellow at the University of Oklahoma.
### My GitHub has a few repositories, mostly dealing with methods for processing raw metagenomic sequencing and analyzing the resulting data. Some examples:

#### The latest addition: **fun-with-ggplots**

Description: Some visualizations using ggplot. These started to get fairly complex once I layered faceting onto significance bars, onto grid.arrange, etc. So, this is mostly a repository of information I don't want to forget

<pre>
<img src=https://github.com/kameron-sugino/fun_with_ggplots/blob/main/fun_with_ggplots_files/figure-markdown_github/unnamed-chunk-16-1.png
 alt="drawing" width="450"/> 
</pre>

#### **Basics-of-16S-sequence-processing-with-QIIME-on-SGE-cluster-servers**

Description: This is a primer for processing 16S sequence data on a cluster server, using the LSF platform as the example. We go over formatting a batch job, submitting a batch job, checking the status of jobs, and cancelling a job. Within QIIME, we will QC and trim the reads, create OTU tables, and annotate the taxonomic IDs using greengenes2.

#### **Basics-of-Microbiome-Data-Analysis:**

Description: A primer on basic microbiota analysis methods geared towards anyone who is new to R. We go over basic R functions, rarefaction of microbiota count data, alpha diversity, beta diversity, negative binomial regression, as well as the associated plots, tables, and statistics.

<pre>
<img src=https://github.com/kameron-sugino/Basics-of-Microbiome-Data-Analysis/blob/main/20200327_V1.3_Workshop_Code_ARCHBG_Markdown_files/figure-markdown_github/plot-rarefaction-curve-1.png alt="drawing" width="450"/> 
Figure 1: Rarefaction plot showing the depth of coverage after subsampling an microbiota count data from an OTU table.


<img src=https://github.com/kameron-sugino/Basics-of-Microbiome-Data-Analysis/blob/main/20200327_V1.3_Workshop_Code_ARCHBG_Markdown_files/figure-markdown_github/plot-pcoa-of-paired-samples-1.png alt="drawing" width="450"/> 
Figure 2: Bray-Curtis PCoA plot. Arrows link participant samples from baseline measurement to post-intervention microbiome community structure.


<img src=https://github.com/kameron-sugino/Basics-of-Microbiome-Data-Analysis/blob/main/20200327_V1.3_Workshop_Code_ARCHBG_Markdown_files/figure-markdown_github/bubble-plot-of-wheat-data-1.png alt="drawing" width="600"/>
Figure 3: Bubble plot displaying the change in microbiota relative abundance from baseline to end of intervention for the paried samples shown in figure 2. 
Size of the circle denotes the magnitude of the change in abundance; filled circles are an increase in abundace and empty circles a decrease in abundance compared to baseline.
</pre>
 
#### **More-Microbiome-Analysis-Methods:**

Description: An extension of Basics-of-Microbiome-Data-Analysis. We cover repeated measures regression, basic regression model selection criteria using AICc, PCoA model selection criteria using AICc, variable selection using LASSO regression, and visualize the results.

<pre>
<img src=https://github.com/kameron-sugino/More-Microbiome-Analysis-Methods/blob/main/20230324_Workshopcode_2_files/figure-markdown_github/unnamed-chunk-20-2.png
 alt="drawing" width="1000"/>
Figure 4: Correlation heatmap between gut microbiota abundance vs serum metabolite levels

 
<img src=https://github.com/kameron-sugino/More-Microbiome-Analysis-Methods/blob/main/20230324_Workshopcode_2_files/figure-markdown_github/unnamed-chunk-22-1.png  alt="drawing" width="800"/>
Figure 5: Alluvial plot displaying gut microbiome gene pathways that are positively associated with serum metabolites. 
</pre>
#### **Comparing-probiotic-compositions-to-their-labels:**

Description: A short analysis on the accuracy of probiotic labels versus the measured composition.

<pre>
<img src=https://github.com/kameron-sugino/Comparing-probiotic-compositions-to-their-labels/blob/main/probiotics_plots_files/figure-markdown_github/unnamed-chunk-4-1.png alt="drawing" width="450"/>
Figure 6: Probiotic composition measured by 16S sequencing compared to the claims on probiotic labels.

 
<img src=https://github.com/kameron-sugino/Comparing-probiotic-compositions-to-their-labels/blob/main/probiotics_plots_files/figure-markdown_github/unnamed-chunk-5-1.png alt="drawing" width="450"/>
Figure 7: More granular plot of figure 6, showing the specific bugs on or off label for each product.

 
<img src=https://github.com/kameron-sugino/Comparing-probiotic-compositions-to-their-labels/blob/main/probiotics_plots_files/figure-markdown_github/unnamed-chunk-6-1.png alt="drawing" width="1000"/>
Figure 8: Heatmap plot of the data shown in figure 7
</pre>

#### I also upload side projects. My last one is a cost splitting calculator for a group trip with friends -- it helps make sure eveyone is square!

